client/src/components/Home/index.js

```diff
@@ -0,0 +1,20 @@
+import React from 'react';
+import { Button } from 'antd';
+import './style.css';
+
+function Home() {
+  return (
+    <section className='home'>
+      <h1 className='home--heading'>
```

Comment:
this please for some description(not header)  about the site then use P tag better than H1 (not header)